unix_integration/resolver/src/unix_config.rs

```diff
@@ -69,7 +69,7 @@ struct KanidmConfigV2 {
     conn_timeout: Option<u64>,
     request_timeout: Option<u64>,
     pam_allowed_login_groups: Option<Vec<String>>,
-    extend: Vec<GroupMap>,
+    map_group: Vec<GroupMap>,
```

Comment:
Why isn't this called group_maps or something? 😄 Makes my brain itch...

Reply:
Because of what it becomes in the config - it's a table of 

```
[[kanidm.map_group]]
local: "name"
with: "kanidm group"
```

So it kind of becomes english-like - kanidm maps group "name" with "remote".

That's why I chose it